slack_formatter.py

```diff
@@ -41,15 +42,30 @@ def text_block(self, text, markdown=True):
     def divider(self):
         return { "type": "divider" }
 
-    def simple_comparison(self, cur_item, prev_item, found_prev=True, print_num=True, is_percent=False):
+    def simple_comparison(self, cur_item, prev_item, found_prev=True, print_num=True, is_percent=False, label=None):
```

Comment:
This is quite the magical function.

My recommendation would be to break it into at least two smaller chunks, perhaps keeping the type casting separate from the the string manipulation at the bottom.

Will you also please add a docblock?

Reply:
Thank you! Will do!